health-sdk/sdk/src/main/res/layout/ghs_fragment_review.xml

```diff
@@ -130,7 +140,7 @@
         </com.google.android.material.textfield.TextInputLayout>
 
         <androidx.constraintlayout.widget.Barrier
-            android:id="@+id/iban_amount_barrier"
+            android:id="@+id/iban_amount_barrier_bottom"
```

Comment:
do we want every id to start with ghs? if yes, this and some other need to be modified

Reply:
Oh, thank you for catching this! I will fix the ids when I update the style, color, typography and dimension resources in the layout ([IPC-160](https://ginis.atlassian.net/browse/IPC-160)).